HerPublicWebsite/Views/Questionnaire/Confirmation.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using HerPublicWebsite.Controllers
+
+@model HerPublicWebsite.Models.Questionnaire.ConfirmationViewModel
+
+@{
+    ViewBag.Title = "You may be eligible";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        @await Html.GovUkPanel(new PanelViewModel
+        {
+            TitleText = "Application submitted",
+            Html = @<text>Reference code: <strong>@Model.ReferenceCode</strong></text>
+        })
+    </div>
+</div>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        @if (!string.IsNullOrEmpty(Model.ConfirmationEmailAddress))
+        {
+            <p class="govuk-body">You will receive an email confirmation at <strong>@Model.ConfirmationEmailAddress</strong></p>
+        }
+
+        <h2 class="govuk-heading-l">What happens next</h2>
+
+        <p class="govuk-body"><strong>@Model.LocalAuthorityName council or their official contractor</strong> will process
+            your details and contact you if they need any more information on the contact details you proviced.</p>
+
+        @await Html.GovUkInsetText(new InsetTextViewModel
+               {
+                   Text = "Your local authority or their official contractor will contact you within 10 working days."
+               })
+
+        <p class="govuk-body">To enquire about your application visit <a href="@Model.LocalAuthorityWebsite">@Model.LocalAuthorityName council's website</a>.
+            Where you can find out other ways to improve the energy efficiency of your home.</p>
+
+        <form action="@Url.Action(nameof(QuestionnaireController.Confirmation_Post), "Questionnaire")" method="post" novalidate>
```

Comment:
I think there's an argument whether this is 'confirmation post' or 'future schemes post', but while there's only one button on the page it doesn't matter :)